examples/cpp/infer_multi/infer_multi.cpp

```diff
@@ -0,0 +1,224 @@
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "tensorflow/core/framework/graph.pb.h"
+#include "tensorflow/core/framework/op.h"
+#include "tensorflow/core/framework/tensor_types.h"
+#include "tensorflow/core/graph/algorithm.h"
+#include "tensorflow/core/graph/default_device.h"
+#include "tensorflow/core/graph/graph.h"
+#include "tensorflow/core/graph/graph_constructor.h"
+#include "tensorflow/core/platform/env.h"
+
+#include "tensorflow/cc/client/client_session.h"
+#include "tensorflow/cc/ops/standard_ops.h"
+#include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/public/session.h"
+
+#include "ngraph_backend_manager.h"
+#include "version.h"
+
+using namespace std;
+namespace tf = tensorflow;
+
+extern tf::Status LoadGraph(const string& graph_file_name,
+                            std::unique_ptr<tf::Session>* session,
+                            const tf::SessionOptions& options);
+
+extern tf::Status ReadTensorFromImageFile(const string& file_name,
+                                          const int input_height,
+                                          const int input_width,
+                                          const float input_mean,
+                                          const float input_std,
+                                          std::vector<tf::Tensor>* out_tensors);
+
+extern tf::Status PrintTopLabels(const std::vector<tf::Tensor>& outputs,
+                                 const string& labels_file_name);
+
+// Prints the available backends
+void PrintAvailableBackends() {
+  // Get the list of backends
+  auto supported_backends =
+      tf::ngraph_bridge::BackendManager::GetSupportedBackendNames();
+  vector<string> backends(supported_backends.begin(), supported_backends.end());
+
+  cout << "Available backends: " << endl;
+  for (auto& backend_name : backends) {
+    cout << "Backend: " << backend_name << std::endl;
+  }
+}
+
+// Sets the specified backend. This backend must be set BEFORE running
+// the computation
+tf::Status SetNGraphBackend(const string& backend_name) {
+  // Select a backend
+  tf::Status status =
+      tf::ngraph_bridge::BackendManager::SetBackendName(backend_name);
+  return status;
+}
+
+// Create a simple computation graph and run
+void RunSimpleNetworkExample() {
```

Comment:
May be we can remove this example. It is in examples/tf_cpp_examples/hello_tf.cpp

Reply:
I guess so. Will do.